### ISSUE TITLE
Multiple Key for one user ?

### ISSUE THREAD

**Issue by anthonymartins** [attachmentgenie/attachmentgenie-ssh#51]
Hi,

I am new to Puppet and I use your ssh module, but I can't add several public keys to the same user?

I tried this but it did not work : 

```
ssh::user { 'my_user':
  key => ['akalkalkalka', 'bkblkblkblkb'],
  comment => ['key1', 'key2']
}
```

and if i try this too : 

```
ssh::user { 'my_user':
  key => 'akalkalkalka',
  comment => 'key1',
}

ssh::user { 'my_user':
  key => 'bkblkblkblkb',
  comment => 'key2',
}
```

I have "duplcated content" !

Is it possible to do what I need with your module ?

**Comment by attachmentgenie** [comment]
@anthonymartins Currently you cant using this module.Both code snippets will (unsuccessfully) try to include 2 Ssh::User[my_user] resources. As this defined type uses the name variable to determine which user to add the key for. Which is a fairly simple implementation, but that does mean you can only include 1 key per user.

I the very near future I intend to rectify this short coming, hopefully in a backward compatible way.

**Comment by anthonymartins** [comment]
Thank you :)

Finally i use this : 

```
ssh_authorized_key { 'key1@my_mail':
  ensure => present,
  user => 'my_user',
  type => 'rsa',
  key => 'akalkalkalka',
}

ssh_authorized_key { 'key2@my_mail':
  ensure => present,
  user => 'my_user',
  type => 'rsa',
  key => 'bkblkblkblkb',
}
```

**Comment by attachmentgenie** [comment]
released as part of 2.1.0
